test/Dialect/FIRRTL/emit-basic.mlir

```diff
@@ -704,13 +704,18 @@ firrtl.circuit "Foo" {
     }
   }
   // CHECK:      module ModuleWithGroups :
-  // CHECK-NEXT:   layerblock GroupA :
+  // CHECK-NEXT:   output a : Probe<UInt<1>, GroupA>
+  // CHECK-NEXT:   output b : RWProbe<UInt<1>, GroupA.GroupB>
+  // CHECK:        layerblock GroupA :
   // CHECK-NEXT:     layerblock GroupB :
   // CHECK-NEXT:       layerblock GroupC :
   // CHECK-NEXT:       layerblock GroupD :
   // CHECK-NEXT:         layerblock GroupE :
   // CHECK-NEXT:     layerblock GroupF :
-  firrtl.module @ModuleWithGroups() {
+  firrtl.module @ModuleWithGroups(
+    out %a: !firrtl.probe<uint<1>, @GroupA>,
+    out %b: !firrtl.rwprobe<uint<1>, @GroupA::@GroupB>
+  ) {
     firrtl.layerblock @GroupA {
       firrtl.layerblock @GroupA::@GroupB {
```

Comment:
It could be elided in the printed/parsed representation. The storage should be a `SymbolRefAttr` to make symbol user verification trivial. I've been trying to avoid custom printer/parsers whenever possible as they're a pain to maintain... 😉

Reply:
It can't be elided as long as we want to use a nested (a::b::c) symbol ref attr. I want to leave this as-is, but will keep thinking about whether the underlying storage makes sense.

Reply:
Yeah, I'm good to leave it. 👍